src/data/miscellaneous.csv

```diff
@@ -1,19 +1,19 @@
 "text", "translate"
 
 "161 – Length preference"
-"中",""
-"中～長",""
-"短",""
-"短～中",""
-"長",""
+"短","Short"
+"短～中","Short–Medium"
+"中","Medium"
+"中～長","Medium–Long"
+"長","Long"
```

Comment:
Need to check if this has strict length limit and will display properly

Reply:
where are these in the game? i cant seem to find them

Reply:
Yeah it doesn't seem to be used; removing them